dictionaries/software-terms/cspell-corrections.yaml

```diff
@@ -0,0 +1,50 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/streetsidesoftware/cspell/main/cspell.schema.json
+# cspell:ignoreRegExp /- \w+-/
+# cspell:locale en-us
+
+# Corrections to common spelling mistakes in software terms
```

Comment:
Did you look at codespell ? They have a bunch of this

https://github.com/codespell-project/codespell/blob/master/codespell_lib%2Fdata%2Fdictionary_code.txt

License is GPL2

Reply:
@ccoVeille,

Thank you. Good point, I didn't see their list. 

> License is GPL2

Software terms needs to remain MIT, so using a GPL source is out of the question.

Reply:
Good point

At least, I can use your list in jargonLint 🥰